tools/topology/sof-hda-generic.m4

```diff
@@ -30,11 +30,13 @@ include(`platform/intel/intel-generic-dmic.m4')
 #
 # Define the pipelines
 #
-# PCM0  <---> volume (pipe 1,2) <----> HDA Analog (HDA Analog playback/capture)
-# PCM1  <---> volume (pipe 3,4) <----> HDA Digital (HDA Digital playback/capture)
-# PCM3  ----> volume (pipe 7)   -----> iDisp1 (HDMI/DP playback, BE link 3)
-# PCM4  ----> Volume (pipe 8)   -----> iDisp2 (HDMI/DP playback, BE link 4)
-# PCM5  ----> volume (pipe 9)   -----> iDisp3 (HDMI/DP playback, BE link 5)
+# PCM0P --> volume     (pipe 1) --> HDA Analog (HDA Analog playback/capture)
+# PCM0C <-- volume, EQ (pipe 2) <-- HDA Analog (HDA Analog playback/capture)
+# PCM1P --> volume     (pipe 3) --> HDA Digital (HDA Digital playback/capture)
+# PCM1C <-- volume, EQ (pipe 4) <-- HDA Digital (HDA Digital playback/capture)
```

Comment:
Each of the above 4 pipes is only either playback or capture, but since they are arranged in pairs per PCM, I guess keeping "playback/capture" in comments makes sense.

Reply:
I think I should change the comment also, I didn't notice that.

Reply:
I've fixed it now.